DisguiseUnityRenderStream/Runtime/PipelineAbstraction/CameraCapturePresenter.cs

```diff
@@ -42,7 +46,22 @@ protected override void OnEnable()
         protected override void Update()
         {
             base.Update();
-            
+
+            if (m_cameraCapture.description.m_autoFlipY && autoFlipY)
+            {
+                autoFlipY = false;
+                
+                Debug.LogWarning($"Disabled {nameof(CameraCapturePresenter)}.{nameof(CameraCapturePresenter.autoFlipY)}" +
+                                 $"because it's already enabled in the sibling {nameof(CameraCapture)} component");
+            }
+
+            sourceColorSpace = m_cameraCapture.description.m_colorSpace switch
+            {
+                CameraCaptureDescription.ColorSpace.Linear => SourceColorSpace.Linear,
```

Comment:
Minor / Brainstorm: I've seen mapping between enums like that in a few places, could we use the same enum everywhere (avoiding the need for those enum consntant mapping)?

Reply:
The idea behind it is:

`SRGBConversions` has color space and color conversion enums. Those ones will never change.

`CameraCaptureDescription` has a color space enum and `BlitExtended` has a color conversion enum. These could grow in the future to support more color spaces.